Jetcaster/wear/src/main/java/com/example/jetcaster/ui/library/LibraryScreen.kt

```diff
@@ -0,0 +1,198 @@
+                                downloadItemArtworkPlaceholder = rememberVectorPainter(
+                                    image = Icons.Default.MusicNote,
+                                    tintColor = Color.Blue,
+                                ),
+                                onClick = {
+                                    libraryScreenViewModel.onTogglePodcastFollowed(podcast.uri)
+                                },
+                            )
+                        }
+                    } else {
+                        item {
+                            PlaceholderChip(
+                                contentDescription = "",
+                                colors = ChipDefaults.secondaryChipColors()
+                            )
+                        }
+                    }
+                }
+            }
+
+        is LibraryScreenUiState.Ready ->
+            ScreenScaffold(scrollState = columnState, modifier = modifier) {
+                ScalingLazyColumn(columnState = columnState) {
+                    item {
+                        ResponsiveListHeader(modifier = Modifier.listTextPadding()) {
+                            Text(stringResource(R.string.home_library))
+                        }
+                    }
+                    item {
+                        Chip(
+                            label = stringResource(R.string.latest_episodes),
+                            onClick = onLatestEpisodeClick,
+                            icon = DrawableResPaintable(R.drawable.new_releases),
+                            colors = ChipDefaults.secondaryChipColors()
+                        )
+                    }
+                    item {
+                        Chip(
+                            label = stringResource(R.string.podcasts),
+                            onClick = onYourPodcastClick,
+                            icon = DrawableResPaintable(R.drawable.podcast),
+                            colors = ChipDefaults.secondaryChipColors()
+                        )
+                    }
+                    item {
+                        ResponsiveListHeader(modifier = Modifier.listTextPadding()) {
+                            Text(stringResource(R.string.queue))
+                        }
+                    }
+                    item {
+                        if (s.queue.isEmpty()) {
+                            QueueEmpty()
+                        } else {
+                            Chip(
+                                label = stringResource(R.string.up_next),
+                                onClick = onUpNextClick,
+                                icon = DrawableResPaintable(R.drawable.up_next),
+                                colors = ChipDefaults.secondaryChipColors()
+                            )
+                        }
+                    }
+                }
+            }
+    }
+}
+
+@Composable
+fun LoadingScreen(columnState: ScalingLazyColumnState) {
```

Comment:
This composable (and the ones below) should also accept a modifier.

Reply:
thanks fixed